frontend/src/Editor/LeftSidebar/SidebarPageSelector/PageHandler.jsx

```diff
@@ -137,14 +145,23 @@ export const PageHandler = ({
                 data-cy={'home-page-icon'}
               />
             )}
-            <SortableList.DragHandle show={isHovered} />
-          </div>
-          <div
-            className="col text-truncate font-weight-400 page-name"
-            data-cy={`pages-name-${String(page.name).toLowerCase()}`}
-          >
-            {page.name}
+            <SortableList.DragHandle show={isHovered && !isDisabled} />
           </div>
+          {!isDisabled ? (
```

Comment:
Suggestion: have `disabled` text with a slightly different styling like a tag may be a better UX (How draft is in below image)
<img width="447" alt="image" src="https://github.com/ToolJet/ToolJet/assets/57667706/47ad4af0-55f8-4e07-a6fe-585c6cdfdab1">

Reply:
Or can follow the same UX of indicating with icon like how we do for hide menu.

Reply:
Done as per new design.